FreeAgent.postman_collection.json

```diff
@@ -1159,8 +1150,23 @@
 					},
 					"response": []
 				}
-			],
-			"protocolProfileBehavior": {}
+			]
+		},
+		{
+			"name": "Bill Items",
+			"item": [
+				{
+					"name": "Index",
+					"request": {
+						"method": "GET",
+						"header": [],
+						"url": {
```

Comment:
Looks like the URL and headers are missing here -- feels like maybe the request wasn't saved before you exported the collection? If you see something like the below in your Postman, it needs a Cmd+S to be saved (or clicking on the button next to `Send`)

Reply:
I think that should be fixed now